apps/simulators/src/app/app.component.html

```diff
@@ -1,89 +1,55 @@
 <biosimulations-navigation
-  *ngIf="!isMobileSimulators; else mobileSimulators"
-  [appName]="config.appName"
-  [appNameParts]="config.appNameParts"
-  [logo]="config.logo"
-  [privacyPolicyVersion]="config.privacyPolicyVersion"
-  [newsVersion]="1"
-  [showNews]="false"
-  [healthy]="healthy$ | async">
-  <!-- Start Hero Banner -->
-  <biosimulations-topbar-menu>
-    <biosimulations-hover-open-menu [disabled]="true" [routerLink]="['/simulators']">
-      <biosimulations-topbar-menu-item heading="BioSimulators" icon="simulator" id="trigger" target="_self">
-      </biosimulations-topbar-menu-item>
-    </biosimulations-hover-open-menu>
-
-    <biosimulations-hover-open-menu [disabled]="true" [href]="config.dispatchAppUrl">
-      <biosimulations-topbar-menu-item heading="Run Simulations" icon="experiment" id="trigger" target="_self">
-      </biosimulations-topbar-menu-item>
-    </biosimulations-hover-open-menu>
-
-    <biosimulations-hover-open-menu>
-      <biosimulations-topbar-menu-item heading="Utilities" icon="construction" id="trigger">
-      </biosimulations-topbar-menu-item>
-      <ng-container id="items">
-        <biosimulations-dropdown-menu-item heading="Convert a file" [routerLink]="['/utils', 'convert-file']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Create a project" [routerLink]="['/utils', 'create-project']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate a model" [routerLink]="['/utils', 'validate-model']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item
-          heading="Validate a simulation"
-          [routerLink]="['/utils', 'validate-simulation']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate metadata" [routerLink]="['/utils', 'validate-metadata']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate a project" [routerLink]="['/utils', 'validate-project']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item
-          heading="Validate a simulator"
-          href="https://github.com/biosimulators/Biosimulators/issues/new?assignees=&labels=Validate%2Fsubmit+simulator&template=ValidateOrSubmitASimulator.yml&title=%5BSimulation+capabilities%5D%3A+"
-          target="githubissues">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Suggest a simulator" [routerLink]="['/utils', 'suggest-simulator']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="BioSimulators REST API" [href]="config.apiUrl">
-        </biosimulations-dropdown-menu-item>
-      </ng-container>
-    </biosimulations-hover-open-menu>
-
```

Comment:
removed Utilities navigation from simulators app, is this intentional?  Did move the navigation to the Utilities somewhere else, it seems to be gone from all of the apps ???

Reply:
it shouldn't be too hard to add these links to the appRoutes class